trio/_subprocess.py

```diff
@@ -621,7 +614,14 @@ async def read_output(stream, chunks):
 
                     async def killer():
                         with killer_cscope:
-                            await deliver_cancel(proc)
+                            try:
+                                await deliver_cancel(proc)
+                            except BaseException as exc:
+                                LOGGER = logging.getLogger("trio.run_process")
+                                LOGGER.exception(
+                                    f"tried to kill process {proc!r}, but failed with: {exc!r}"
+                                    )
```

Comment:
Dedent this paren to satisfy black. (It's often easier to run `black setup.py trio` before you commit than it is to predict what black will like.)

This should have failed the formatting check, but it looks like we're not properly rejecting PRs that require black changes. I'll address that in a separate PR.